R/results_table.R

```diff
@@ -31,13 +31,14 @@ get_float_columns <- function(data) {
   return(setdiff(numeric_columns, integer_columns))
 }
 
-#' Create an interactive or static table
+#' Format the signal results to an interactive or static table
 #'
 #' This function creates an interactive DataTable or a static gt table depending
-#' on the value of the `interactive` parameter. It can also format float columns
-#' to have two decimal places.
+#' on the value of the `interactive` parameter. It takes the stratum and category columns and transforms those to be headers of sections containing all signal results for these strata. It colors rows red for tables containing the any_alarms column when there was >= 1 alarm. It makes the table downloadable as CSV, Excel, PDF.
 #'
 #' @param data A data frame.
+#'
+#' @param positive_only Logical indicating whether to filter only those signal results where a signal was generated (default is TRUE). When TRUE then the signals column is not removed from the table.
```

Comment:
The description of positive_only is not super clear to me, Maybe something like: Logical indicating whether to filter the signal results to include only the weeks when a signal was generated (default is TRUE). If set to TRUE, the signals column will not be removed from the table.